### DISCUSSION TITLE
CSS customization

### DISCUSSION THREAD

**Discussion by MakoEOS** [globaleaks/globaleaks-whistleblowing-software#3491] [Q&A]
Hi,
I wanted to ask if there was a css template from which to modify the site's graphics, I have already tried uploading css created from scratch but the changes are not applied

**Comment by evilaliv3** [comment]
Thank you for your questions @MakoEOS 

Are you using a multitenant setup or a single tenant setup?

**Reply by MakoEOS** [reply]
Hi @evilaliv3,
I'm using a single tenant setup.

**Comment by evilaliv3** [comment]
Can you show an example of configuration that you are trying to make and that is not working?

**Reply by MakoEOS** [reply]
@evilaliv3 
now I'm trying this to change backgroung color and the alignament of the the logo

`#HeaderBox {
  color: #ffffff;
  background-color: #ff8800;
}

#ProjectTitle, .TitleSeparator {
  display: none;
}


element.style {
}
#LogoBox img {
    max-height: 10rem;
    max-width: 20rem;
}
[dir] img {
    border-style: none;
}
img {
    vertical-align: middle;
}`

the background color change, but the alignament of the logo dont change

**Comment by flashlight4** [comment]
For me, this CSS-Snippet worked to center the logo fairly good.
`#HeaderBox {
  display: flex;
  flex-direction: column; /* Inhalt vertikal anordnen */
  justify-content: center; /* Vertikal zentrieren */
  align-items: center; /* Horizontal zentrieren */
}`

It's not suitable to 100 % while there are two header boxes.
